src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -893,12 +898,20 @@ public static void setup(SlimefunPlugin plugin) {
 		.register(plugin);
 
 		new SlimefunArmorPiece(categories.armor, SlimefunItems.HAZMATSUIT_CHESTPLATE, RecipeType.ARMOR_FORGE,
-		new ItemStack[] {new ItemStack(Material.ORANGE_WOOL), null, new ItemStack(Material.ORANGE_WOOL), new ItemStack(Material.ORANGE_WOOL), new ItemStack(Material.ORANGE_WOOL), new ItemStack(Material.ORANGE_WOOL), new ItemStack(Material.BLACK_WOOL), new ItemStack(Material.BLACK_WOOL), new ItemStack(Material.BLACK_WOOL)},
+		new ItemStack[] {
+		    SlimefunItems.LEAD_PLATE, null, SlimefunItems.LEAD_PLATE,
+			SlimefunItems.LEAD_PLATE, new ItemStack(Material.ORANGE_WOOL), SlimefunItems.LEAD_PLATE,
+			SlimefunItems.LEAD_PLATE, new ItemStack(Material.BLACK_WOOL), SlimefunItems.LEAD_PLATE
```

Comment:
I find this one quite extreme, I would probably suggest to reduce it to 2 plates per armorpiece.

Reply:
Eh, it's still pretty easy to make

Reply:
I think it's still pretty easy to make. It's supposed to be protective, should be "quite extreme"